extensions/fop/runtime/src/main/resources/META-INF/native-image/proxy-config.json

```diff
@@ -0,0 +1,3 @@
+[
+  ["org.apache.fop.render.RendererEventProducer"]
```

Comment:
Could `NativeImageProxyDefinitionBuildItem` be used here instead of `proxy-config.json`.

Reply:
@jamesnetherton I was looking for a build Item, with this functionality, I'll refactor it.

Reply:
Done

Reply:
@JiriOndrusek you may want to upvote https://github.com/quarkusio/quarkus/issues/9945 :)